frontend/public/locales/fr/renew-adult-child.json

```diff
@@ -1,4 +1,20 @@
 {
+  "confirm-marital-status": {
+    "page-title": "Marital-status",
+    "has-marital-status-changed": "Has your marital status or spouse changed since you applied for the Canadian Dental Care Plan?",
+    "help-message": "The marital status we have on file can also be found on the renewal letter you received from Service Canada.",
+    "back-btn": "Retour",
+    "continue-btn": "Continuer",
+    "cancel-btn": "Annuler",
+    "save-btn": "Sauvegarder",
+    "radio-options": {
+      "yes": "Oui",
+      "no": "Non"
+    },
+    "error-message": {
+      "has-marital-status-changed-required": "Please indicate if your marital status has changed"
+    }
+  },
```

Comment:
French text can be found in Figma: https://www.figma.com/design/UfB3nkCLGIwMH1two1M1RN/CDCP-Renewals?node-id=2566-14998&node-type=canvas&t=f9hEumXp9YH44s5C-0

The error message will also have to be translated. Tagging @nibivi77. In the meantime the value can be marked with "[FR]".

Reply:
The French Figma does not thave the French content for Confirm Marital status screen. That screen is Address, maybe it's used as a placeholder

Reply:
Good catch - resolving.